rule207.md

```diff
@@ -7,7 +7,13 @@ Type: Mutable
 
 # Rule
 
-Each player always has exactly one vote.
+Each player always has exactly one vote. They can change their vote on a proposal as many times as they would like. When quorum is reached, the Player's most recent vote will be counted.
```

Comment:
>They can change their vote on a proposal as many times as they would like.

Maybe we can refactor this sentence to define how votes are counted instead of how players can cast votes. Something like:

    The Player's vote is the last `+1`/`-1` comment they left on a rule-change thread, once the final form of the proposal has been decided and voting has begun.